<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog2</class>
 <widget class="QDialog" name="Dialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>施工坐标与测量坐标转换</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>测量坐标原点x₀(m)</string>
         </property>
         <property name="buddy">
          <cstring>x0Edit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>测量坐标原点y₀(m)</string>
         </property>
         <property name="buddy">
          <cstring>y0Edit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>施工坐标A₁(m)</string>
         </property>
         <property name="buddy">
          <cstring>A1Edit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>施工坐标B₁(m)</string>
         </property>
         <property name="buddy">
          <cstring>B1Edit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="text">
          <string>施工坐标系纵轴方向角α(angle)</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="buddy">
          <cstring>alphaEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>测量坐标x₁(m)</string>
         </property>
         <property name="buddy">
          <cstring>x1Edit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>测量坐标y₁(m)</string>
         </property>
         <property name="buddy">
          <cstring>y1Edit</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="x0Edit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="y0Edit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="A1Edit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="B1Edit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="alphaEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="x1Edit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="y1Edit"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="runButton">
         <property name="text">
          <string>计算</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>x0Edit</tabstop>
  <tabstop>y0Edit</tabstop>
  <tabstop>A1Edit</tabstop>
  <tabstop>B1Edit</tabstop>
  <tabstop>alphaEdit</tabstop>
  <tabstop>x1Edit</tabstop>
  <tabstop>y1Edit</tabstop>
  <tabstop>runButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
